content/en/seachat/seachat-manual/02-create-agent/02-live-agent-transfer.md

```diff
@@ -45,6 +45,10 @@ If a customer requests a live agent during the chat conversation, the human agen
 <br/>
 <br/>
 
+> **Agent Conversations**
```

Comment:
hmm this is not what I had in mind for writing about MD support:

1. you should create a new section specifically called "Markdown Support"
2. the meat is in the sentence "Both your live agents and your users can insert links and Markdown text in the chat conversation" -- but easy to miss when reading it:
a. 2 pictures should be given: one is human agent UI, one is chat UI
b. emphasize that it's only supported in web UI, not line/whatsapp etc. The statement above didn't do this.